backend/prisma/schema.prisma

```diff
@@ -43,13 +43,13 @@ model Transaction {
 }
 
 model Budget {
-  id        String   @id @default(cuid())
-  userId    String
-  amount    Float
-  weekStart DateTime
-  createdAt DateTime @default(now())
-  updatedAt DateTime @updatedAt
-  user      User     @relation(fields: [userId], references: [id])
+  id         String   @id @default(cuid())
+  userId     String
+  amount     Float
+  monthStart DateTime?
```

Comment:
why mark this field optional? I imagine every budget entry to have corresponding month 😄

If you need to backfill old data, consider a migration script.